donau-universitat-krems-zentrum-fur-e-governance.csl

```diff
@@ -0,0 +1,1335 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" page-range-format="chicago" default-locale="de-AT" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Donau-Universität Krems - Zentrum für E-Governance</title>
```

Comment:
@POBrien333, shouldn't this be for the "Department für E-Governance in Wirtschaft und Verwaltung"? (per https://forums.zotero.org/discussion/comment/294010/#Comment_294010)

Also, the title should include " (German - Austria)" if the locale remains "de-AT".